### ISSUE TITLE
Create subdomain for production instance

### ISSUE THREAD

**Issue by jaythomas** [JTCC-Programming-Club/weather-station#17]
We need a production server and subdomain to host the API server on once it is complete. Somewhere that can handle a growing PostgreSQL database and provide direct shell access for maintenance and remote debugging.

**Comment by jaythomas** [comment]
This is live and deployed and will be manually updated from from the `api` branch until merged into `master`.
